cli/azd/cmd/provision.go

```diff
@@ -43,7 +43,7 @@ func (i *provisionFlags) bindCommon(local *pflag.FlagSet, global *internal.Globa
 	local.BoolVar(&i.preview, "preview", false, "Preview changes to Azure resources.")
 	local.BoolVar(
 		&i.ignoreDeploymentState,
-		"disable-deployment-state",
+		"ignore-deployment-state",
```

Comment:
@savannahostrowski - Thoughts on what this flag name should be?  I'm thinking something short and sweet like --ignore-state, but more verbose is fine too.

Reply:
I thought we were using `--no-cache`?

Reply:
updated to `no-state`
Talked offline to Savannah about why I would prefer `state` better than `cache`

Reply:
Please ignore my feedback about this in my review, I did not notice this conversation when I wrote it :-).